src/ components/TableBody/TableRow/index.js

```diff
@@ -0,0 +1,46 @@
+import React from "react"
+import "./TableRow.scss"
+import SlidingSubmenu from "../SlidingSubmenu/"
+import { CSSTransitionGroup } from "react-transition-group"
+import iconAuthor from "../../../images/kara.png"
```

Comment:
It should be `iconAuthor` or `icon-author`, don't know what `kara` means 😄 
```suggestion
import iconAuthor from "../../../images/iconAuthor.png"
```